### ISSUE TITLE
Add a .devcontainer to the project

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/RadioLogger#71]
I think that this project would be great with a .devcontainer 

Add a .devcontainer folder to this project

Microsoft has documented setting up this on their site for Visual Studio/Visual Studio Code.

**Comment by Ferotech-2022** [comment]
okay that would be done

**Comment by pulkitxm** [comment]
Can I get assigned? let me know how to start.

**Comment by gbowne1** [comment]
Yes. I can assign you.  @Pulkitxm Are you familiar with this task?  

It looks like you asked to be assigned other tasks as well.

There is instructions on how to do this task on this page: 
https://code.visualstudio.com/docs/devcontainers/create-dev-container
